providers/redis/redis.go

```diff
@@ -109,7 +121,12 @@ func (p *RedisProvider) DeleteCacheParameterGroup(ctx context.Context, instanceI
 	return err
 }
 
-func (p *RedisProvider) Update(ctx context.Context, instanceID string, params providers.UpdateParameters) error {
+func (p *RedisProvider) UpdateReplicationGroup(ctx context.Context, instanceID string, params providers.UpdateReplicationGroupParameters) error {
+	replicationGroupID := GenerateReplicationGroupName(instanceID)
+	return p.modifyReplicationGroup(ctx, replicationGroupID, params.PreferredMaintenanceWindow)
+}
```

Comment:
I don't think any unit tests cover this.

Reply:
they do now!